api/src/main/resources/messages.properties

```diff
@@ -1927,6 +1927,7 @@ PersonAttributeType.CivilStatus=Civil Status
 PersonAttributeType.HealthDistrict=Health District
 PersonAttributeType.Race=Race
 
+Privilege.required= Required Privileges
```

Comment:
I would make this message `Role.you.must.have.privileges=You must have the following privileges in order to assign them: {0}`